src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -47,7 +46,6 @@ export default class ServiceGraphPage extends React.Component<ServiceGraphPagePr
       isReady: false,
```

Comment:
Not something you did, but 'isReady: true' should be added to setState in onReady(), please add this.

Reply:
Of course. I added it in handleReady() which is the function for onReady event.